neural_mfg.py

```diff
@@ -32,19 +32,25 @@ def __init__(self):
         
         self.DTYPE = 'float32'
         self.var = var
-       
+        
         # MFG parameters 
         self.xi    = var['mfg_params']['xi']
         self.c_s   = var['mfg_params']['c_s']
         self.mu    = var['mfg_params']['mu']
         self.m_0  = var['room']['m_0']
         self.g     = -(2*self.xi**2)/self.m_0
         self.sigma = np.sqrt(2*self.xi*self.c_s)
-        self.l     = -self.g*self.m_0
         self.R     = 0.37
         self.s     =  tf.constant([0, -var['room']['s']],  dtype=self.DTYPE, shape=(1, 2))
         self.pot   = var['mfg_params']['V']
         
+        self.gamma = var['mfg_params']['gamma']; # discount factor
+        self.alpha = var['mfg_params']['alpha']; # conjestion operator
+        if gamma > 0:
+            self.l     = -self.g*self.m_0
+        else:
+            self.l = 0
+        
```

Comment:
Qui hai la condizione per gamma e lambda. Ho aggiunto due parametri da includere nel .config per il discount factor e la congestione (che al momento stiamo considerando 0)

Reply:
I think it is the inverse (here you write when $\gamma > 0$ $\lambda = -gm_0$)

Reply:
Changed it now